ts/components/conversation/ConversationHeader.tsx

```diff
@@ -679,7 +679,7 @@ export class ConversationHeader extends React.Component<PropsType, StateType> {
     );
   }
 
-  private renderHeader(): ReactNode {
+  private getActionForType() {
```

Comment:
Combined with the change to `renderAvatar()`, we would not need to recreate the `onClick` handler.
In that case it's ok to revert this change and keep `onClick` generation as part of `renderHeader()`.

(As a side note, if hypothetically we wanted to keep this new onClick generator, it's maybe more clear to name it verbosely such as `getHeaderOnClickHandler()` because "getActionForType" can be confusing in a complex component.)